dev/ci/README.md

```diff
@@ -172,6 +172,11 @@ If your repository has access to runners tagged `windows`, setting the
 secret variable `WINDOWS` to `enabled` will add jobs building Windows
 versions of Coq (32bit and 64bit).
 
+If the secret variable `WINDOWS` is set to `enabled_all_addons`,
+an extended set of addons will be added to the Windows installer.
+This leads to a considerable runtime in CI so this is not enabled
+by default for pipelines for pull requests.
```

Comment:
In v8.9 this is actually not true: the extended set of addons is built in all cases.
See: https://github.com/coq/coq/blob/v8.9/dev/ci/gitlab.bat